packages/expo-app-auth/ios/EXAppAuth/EXAppAuth+JSON.m

```diff
@@ -19,13 +19,14 @@ + (NSString *)dateNativeToJSON:(NSDate *)input
 + (NSDictionary *)tokenResponseNativeToJSON:(OIDTokenResponse *)input
 {
   if (!input) return nil;
-  
+
   return @{
            @"accessToken": EXnullIfEmpty([input accessToken]),
            @"accessTokenExpirationDate": EXNullIfNil([[self class] dateNativeToJSON:[input accessTokenExpirationDate]]),
            @"additionalParameters": EXNullIfNil([input additionalParameters]),
            @"idToken": EXnullIfEmpty([input idToken]),
            @"tokenType": EXnullIfEmpty([input tokenType]),
+           @"refreshToken": EXnullIfEmpty([input refreshToken]),
```

Comment:
Thanks for opening this PR and getting back to me! 

I'm not sure it is being set, though. That seemed like part of the problem in the first place.  The value was not added to the NSDictionary to begin with, so [this line](https://github.com/mkuipers/expo/blob/fix-for-google-auth-reversed-refresh-token/packages/expo-app-auth/ios/EXAppAuth/EXAppAuth.m#L166) was the `refreshToken` it to be equal to the  `accessToken`

Objective-C is not a language with which I'm super familiar, so it's more likely I'm just missing something.

Reply:
Hmm - though I think I'm understanding [this conditiona](https://github.com/mkuipers/expo/blob/fix-for-google-auth-reversed-refresh-token/packages/expo-app-auth/ios/EXAppAuth/EXAppAuth.m#L166-L170)l a little more now that I cleared up some of my initial confusion.  It looks like it's pulling the `refreshToken` from the `request` object if it's not set on the input. If it is already provided by the `input`, it should just use that value. It looked like it was using `accessToken` instead of `refreshToken` for the assignment on line 169.  I pushed up a fix for that in d4bf549.